velox/exec/tests/WindowTest.cpp

```diff
@@ -79,6 +80,251 @@ TEST_F(WindowTest, spill) {
   ASSERT_GT(stats.spilledPartitions, 0);
 }
 
+TEST_F(WindowTest, rowBasedStreamingWindowOOM) {
+  const vector_size_t size = 100'000;
+  auto data = makeRowVector(
+      {"d", "p", "s"},
+      {
+          // Payload.
+          makeFlatVector<int64_t>(size, [](auto row) { return row; }),
+          // Partition key.
+          makeFlatVector<int16_t>(size, [](auto row) { return row; }),
+          // Sorting key.
+          makeFlatVector<int32_t>(size, [](auto row) { return row; }),
+      });
+
+  createDuckDbTable({data});
+
+  // Abstract the common values vector split.
+  auto valuesSplit = split(data, 10);
+
+  auto planNodeIdGenerator = std::make_shared<core::PlanNodeIdGenerator>();
+  CursorParameters params;
+  auto queryCtx = core::QueryCtx::create(executor_.get());
+  queryCtx->testingOverrideMemoryPool(memory::memoryManager()->addRootPool(
+      queryCtx->queryId(), 2418624, exec::MemoryReclaimer::create()));
```

Comment:
2418624 is 24MB? Can we write as 24 MB? Thanks!

Reply:
@xiaoxmeng It seems 2418624 is 2.31MB not 24MB. Also the type of maxCapacity is int 64 [here](https://github.com/facebookincubator/velox/blob/main/velox/common/memory/Memory.cpp#L247).